addons/website/views/snippets/s_whatsapp.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+	<template id="s_whatsapp" name="WhatsApp Snippet">
+        <section class="s_whatsapp o_not_editable">
+            <div class="s_whatsapp_container position-fixed bottom-0 mb-3">
+                <div class="wa-icon-wrapper position-relative d-inline-flex">
+                    <i class="fa fa-whatsapp rounded-circle d-flex align-items-center justify-content-center shadow-lg o_pos_right" id="wa-fab"></i>
+                    <span class="notification-badge position-absolute rounded-circle"></span>
+                </div>
+                <div class="chatbox bg-white rounded-3 position-absolute shadow-lg overflow-hidden mb-2 d-none">
+                    <div class="header d-flex align-items-center p-2 text-white">
```

Comment:
Add text-white to agent info only not the whole div it will make icon invisible as background itself is white.